### PR TITLE
Add panel for DebugKit to show trashed models

### PR DESCRIPTION
**Pull request by zikani03**
Added a DebugKit Panel to show the number of records trashed using the behavior - useful for debugging

### CODE REVIEW

File: src/Panel/TrashPanel.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ *
+ * Lists how many records were `Trashed` from the configured tables  
+ * 
+ */
+class TrashPanel extends DebugPanel
+{
+    /**
+     * @property {array} Array containing names of tables to check for trashed records
+     */
+    private $_tables = [];
```
Comment (ADmad) [inline src/Panel/TrashPanel.php]: Please use protected instead of private.

File: src/Panel/TrashPanel.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+/**
+ * TrashPanel
+ *
+ * Lists how many records were `Trashed` from the configured tables  
+ * 
+ */
+class TrashPanel extends DebugPanel
+{
+    /**
+     * @property {array} Array containing names of tables to check for trashed records
```
Comment (ADmad) [inline src/Panel/TrashPanel.php]: Remove the curly braces around `array` here and all other types in other methods.

**Comment by ADmad** [comment]
Coding standard error need to be fixed https://travis-ci.org/UseMuffin/Trash/jobs/342397119#L596.
Tests and related template for display needed.

**Comment by davidyell** [comment]
Be nice to get this into the plugin, seems like it would make a good addition. What is outstanding, other than the merge conflicts?

**Comment by davidyell** [comment]
@zikani03 Any progress on this?

**Comment by zikani03** [comment]
@davidyell Apologies for the delay on this, slipped off my radar 😓 . Will rebase and push the suggested changes for review so we can get this closed

**Comment by ADmad** [comment]
Closing due to lack of follow up.

**Pull request closed without merging**